ui/src/app/shared/search-multi-select/search-multi-select.component.html

```diff
@@ -24,7 +24,7 @@
           >
           <div class="m-search-x-controls">
             <div class="m-search-x-clear">
-              <button class="m-iconInteractive" type="button" (click)="clearField()" *ngIf="inputFc.value.length > 0">
+              <button class="m-iconInteractive" type="button" (click)="clearField()" *ngIf="inputFc.value?.length > 0">
```

Comment:
Please use the non-null operator ! instead of the optional ? operator. This will ensure "value" is forced to be defined prior to the comparison.
`*ngIf="inputFc.value!.length > 0"`
https://blog.logrocket.com/understanding-exclamation-mark-typescript/